core/src/protobuf.py

```diff
@@ -181,26 +187,26 @@ def readinto(self, buf: bytearray) -> int:
 
 if False:
     MessageTypeDef = Union[
-        Type[UVarintType],
-        Type[SVarintType],
-        Type[BoolType],
+        type[UVarintType],
+        type[SVarintType],
+        type[BoolType],
         EnumType,
-        Type[BytesType],
-        Type[UnicodeType],
-        Type[MessageType],
+        type[BytesType],
+        type[UnicodeType],
+        type[MessageType],
     ]
-    FieldDef = Tuple[str, MessageTypeDef, Any]
-    FieldDict = Dict[int, FieldDef]
+    FieldDef = tuple[str, MessageTypeDef, Any]
```

Comment:
why does this work here but not in other places?
why is the above `Union` but has `type`s inside?

Reply:
ah i see the updated PR description